endpoints/openrtb2/auction.go

```diff
@@ -1296,8 +1296,14 @@ func (deps *endpointDeps) validateUser(req *openrtb_ext.RequestWrapper, aliases
 	// Check Universal User ID
 	eids := userExt.GetEid()
 	if eids != nil {
-		eidsValue := *eids
-		for eidIndex, eid := range eidsValue {
+
+		validEids, eidErrors := validateEIDs(*eids)
+
+		if len(eidErrors) > 0 {
+			errL = append(errL, eidErrors...)
+		}
```

Comment:
Likely fine to keep this logic here for now. We want to separate validation from fixing in the future. We can refactor this new logic along with the rest later. Thoughts @bsardo?